manifests/kustomize/base/cache/cache-deployment.yaml

```diff
@@ -59,6 +59,23 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: MINIO_SERVICE_SERVICE_HOST
```

Comment:
here the same consistency issues as discussed above arise.

Reply:
ok